integration/bin/tachyon-worker-yarn.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+SCRIPT_DIR="$(cd "$(dirname "$0")"; pwd)"
+source "${SCRIPT_DIR}/common.sh"
+TACHYON_WORKER_JAVA_OPTS="${TACHYON_WORKER_JAVA_OPTS:-${TACHYON_JAVA_OPTS}}"
+
+echo Mount ramdisk
```

Comment:
It makes sense to do the same in tachyon-worker-mesos.sh (and remove the explicit mounting from vagrant).